### DISCUSSION TITLE
Project maintenance

### DISCUSSION THREAD

**Discussion by RosilBelli** [DhiWise/dhiwise-nodejs#39] [General]
Hi!
Is this project in maintenance? 
Can it be used in production?

Thanks!!

**Comment by saloni137** [comment]
Yes, this project is maintained. You can use it in production.

**Reply by proop** [reply]
I'm trying to run the project. I think the dependencies are outdated, because a lot of errors about them when trying to start.
Various (unsuccesfull) results depending on the node/npm version (using nvm to switch between versions 12,14,16,18 and 20).

are there any plans on updating the project ?
